core/src/functions/content.functions.ts

```diff
@@ -281,8 +290,18 @@ export const generateBibleDocument = (
             .sort()
             .map(key => +key)) {
             // look for the section where the phrase is in (if any) and open it if necessary
-            const section = context[level].includedSections.find(
-                _section => phrase.id >= _section.phraseStartId && phrase.id <= _section.phraseEndId
+            const section = context[level].startingSections.find(
```

Comment:
it would be a nice touch to put this .find in a descriptive function

Reply:
and then have a unit test for that function

Reply:
i see what you mean. however in that particular case adding a dedicated function could make the code even more complicated since this method would need 5 or 6 parameters.. i would suggest to only do that in case of redundancy

Reply:
ok, that makes sense